content/v2.0/account-management/pricing-plans.md

```diff
@@ -69,3 +70,9 @@ To protect against any intentional or unintentional harm, Usage-based Plans incl
   - Unlimited PagerDuty, Slack, and HTTP notification endpoints
 
 _To request higher rate limits, contact [InfluxData Support](mailto:support@influxdata.com)._
+
+{{% cloud %}}
+#### VPC and other plans
+
+If you are interested in adding virtual private cloud (VPC) peering to your Cloud account, please [contact sales](mailto:sales@influxdata.com) to discuss the options and associated pricing.
```

Comment:
Don't use the sales email address. Send them to the Contact Sales page: https://www.influxdata.com/contact-sales/

Marketing has had us change a few of these already.